lib/src/cached_package.dart

```diff
@@ -3,23 +3,14 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'package:path/path.dart' as p;
-import 'package:pub_semver/pub_semver.dart';
-import 'package:yaml/yaml.dart';
 
-import 'barback/transformer_config.dart';
-import 'compiler.dart';
-import 'feature.dart';
 import 'io.dart';
 import 'package.dart';
-import 'package_name.dart';
-import 'pubspec.dart';
 
 /// A [Package] whose `lib` directory has been precompiled and cached.
 ///
 /// When users of this class request path information about files that are
-/// cached, this returns the cached information. It also wraps the package's
-/// pubspec to report no transformers, since the transformations have all been
-/// applied already.
+/// cached, this returns the cached information.
 class CachedPackage extends Package {
```

Comment:
This shouldn't be necessary anymore—there's nothing for pub to precompile in the lib directory.

Reply:
Done